cp-ksqldb-server/Dockerfile.ubi8

```diff
@@ -66,30 +54,24 @@ RUN echo "===> Installing Confluent security plugins ..." \
     # We are installing confluent-telemetry package explicitly because
     # KSQL's deb/rpm packages cannot directly depend on
     # confluent-telemetry package as KSQL is Open Source.
-    && yum install -y confluent-telemetry-${CONFLUENT_VERSION}
-
-RUN echo "===> HACK to make cc-role-definitions available in docker image..." \
-    && yum install -y yum-utils \
-    && yum install -y cpio \
-    && cd /tmp \
-    && yumdownloader confluent-ksqldb --resolve \
-    && CONFLUENT_KSQLDB_RPM=$(ls /tmp/confluent-ksqldb-*.rpm) \
-    && echo "===> Extracting the rpm ${CONFLUENT_KSQLDB_RPM} to get the cc-role-definitions jar" \
-    && mkdir /tmp/extracted-ksqldb \
-    && cd /tmp/extracted-ksqldb \
-    && rpm2cpio ${CONFLUENT_KSQLDB_RPM} | cpio -idmv \
-    && cp /tmp/extracted-ksqldb/usr/share/java/ksqldb/cc-role-definitions*.jar /usr/share/java/${COMPONENT}/ \
-    && echo "====> Removing the tmp files" \
-    && rm -rf /tmp/extracted-ksqldb ${CONFLUENT_KSQLDB_RPM} \
-    && chown appuser:appuser /usr/share/java/${COMPONENT}/cc-role-definitions*.jar \
-    && echo "===> Cleaning up ..."  \
+    && yum install -y confluent-telemetry-${CONFLUENT_VERSION} \
+    && echo "====> Installing ksqldb" \
+    && yum install -y confluent-ksqldb-${CONFLUENT_VERSION}
+
+RUN echo "===> Cleaning up ..."  \
     && yum clean all \
     && rm -rf /tmp/* /etc/yum.repos.d/confluent.repo \
     && mkdir -p /usr/share/confluent-hub-components \
     && chown appuser:appuser -R /usr/share/confluent-hub-components \
     && mkdir -p /var/lib/kafka-streams \
     && chown appuser:appuser -R /var/lib/kafka-streams
 
+COPY --chown=appuser:appuser include/etc/confluent/docker/* /etc/confluent/docker/
```

Comment:
if these are static commands, should they be pulled in upper layer?

Reply:
Docker build is a layered approach. Each stmt is a layer being executed after other. In a clean environment, moving it up or down won't make a difference. In local environment, caching will be leveraged for unchanged layers.

Given the time taken by prev stmt is higher, I moved it down. I was making lot of changes in include/etc/confluent/docker/* and it being up was slowing down the debugging for me.

Reply:
I tagged this comment on the wrong line. I meant the following commands are static and they can be moved in upper layers.

```
RUN mkdir -p /etc/${COMPONENT} /etc/${COMPONENT}/secrets /usr/logs \
  && chown appuser:root -R /etc/${COMPONENT} /usr/logs \
  && chmod ug+w -R /etc/${COMPONENT} /usr/logs

RUN chmod +x /etc/confluent/docker/run
RUN chmod +x /etc/confluent/docker/configure
RUN chmod +x /etc/confluent/docker/launch
RUN chmod +x /etc/confluent/docker/ensure
```